src/Files.App/Services/PreviewPopupProviders/QuickLookProvider.cs

```diff
@@ -16,6 +17,7 @@ public sealed class QuickLookProvider : IPreviewPopupProvider
 		private static string pipeName = $"QuickLook.App.Pipe.{WindowsIdentity.GetCurrent().User?.Value}";
 		private static string pipeMessageSwitch = "QuickLook.App.PipeMessages.Switch";
 		private static string pipeMessageToggle = "QuickLook.App.PipeMessages.Toggle";
+		private static Encoding encoding = Encoding.GetEncoding("UTF-8", new EncoderReplacementFallback("?"), new DecoderExceptionFallback());
```

Comment:
What's the reason why this is put in class fields instead of being a local variable?

Reply:
Do you prefer a local variable?

Reply:
Don't we have to put this in L61 as well?
Unless, yes.

Reply:
> Don't we have to put this in L61 as well?

It doesn't look like it, the message we send on L62 is very specific and shouldn't have issues. The message on line 42 can have issues though because it can change based on a number of factors.